frontend/packages/kubevirt-plugin/src/components/vm-disks/table-filters.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from 'lodash';
+import { RowFilter } from '@console/internal/components/filter-toolbar';
+import { DiskType } from '../../constants/vm/storage';
+
+const sourceReducer = (obj) => {
+  const source = obj?.source;
+  const type = obj?.metadata?.type;
+
+  if (type === DiskType.CDROM) {
+    return 'cdrom';
+  }
+
+  if (['Config Map', 'Secret', 'Service Account'].includes(source)) {
```

Comment:
IMO this is too specific and there is no semantic like this in kubevirt API. Can we filter just disk x cdrom x lun ?

And then we could have a 2nd category which would filter by Source

Reply:
+1, only filtering by type now, removed this part of the code.